pkg/identity/identity.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package identity
+
+import (
+	"context"
+
+	"github.com/container-storage-interface/spec/lib/go/csi"
+)
+
+func NewIdentityServer(ident, version string, manifest map[string]string) (csi.IdentityServer, error) {
+	return &IdentityServer{
+		Identity: ident,
+		Version:  version,
+		Manifest: manifest,
+	}, nil
+}
+
+type IdentityServer struct {
+	Identity string
+	Version  string
+	Manifest map[string]string
+}
+
+func (i *IdentityServer) GetPluginInfo(ctx context.Context, req *csi.GetPluginInfoRequest) (*csi.GetPluginInfoResponse, error) {
+	return &csi.GetPluginInfoResponse{}, nil
```

Comment:
Instead of stub results, consider `IdentityServer` to 'inherit' from `csi.UnimplementedIdentityServer`.

Reply:
For the `IdentityServer`, I would actually prefer to do it this way. Most of the logic is already outlined in another commit, and when I open the second PR to add that here, we might as well try to keep the diff as small as possible.